### DISCUSSION TITLE
Allow using variables in "Filter by data values" transformation for repeated rows

### DISCUSSION THREAD

**Discussion by ChrisStrykesAgain** [grafana/grafana#40486] [Dashboards, Panels & Visualizations]
I think this is a feature request: I am trying to come up with an approach that allows me to do a single query, then filter it out by row. For example, I have a query that returns a long list of tests, each of which fulfills a requirement. I have a variable that has a value corresponding to each requirement. 

There are one or more tests for each requirement. My goal is to create a series of rows, one per requirement, that displays only those tests which correspond to the variable value that describes that row. I tried Using a "Filter by value" and using my $requirements variable, but that doesn't seem to work:

![image](https://user-images.githubusercontent.com/30416097/137364998-271e2e46-6f25-4318-9045-70f8bbc36eea.png)

Apologies if this isn't a feature request and I'm missing something obvious.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
